.gitignore

```diff
@@ -3,9 +3,13 @@
 
 ### Files private to developers
 
+# Files that should be git-ignored, but are hand-edited or otherwise valued,
+# and so should not be destroyed by "make clean".
+# start-noclean
 requirements/private.txt
 lms/envs/private.py
 cms/envs/private.py
+# end-noclean
```

Comment:
Do we need to do anything for people that use IDEs? I am unfamiliar with them, but I have a vague remembrance that PyCharm puts dotfiles in your folders or something like that.